lib/mergeASRouterData.js

```diff
@@ -0,0 +1,33 @@
+const fs = require("fs");
+
+// NOTE: Add release versions here in descending order for all the JSON data
+// that is available in `asrouter-local-prod-messages`
+const availableReleases = ["129", "128", "127", "126", "125", "124", "123"];
+
+availableReleases.map((release) => {
+  // Existing message data
+  let result = fs.readFileSync(
+    "lib/asrouter-local-prod-messages/data.json",
+    "utf8",
+  );
+  let json_result = JSON.parse(result);
+
+  // Release message data
+  let data = fs.readFileSync(
+    `lib/asrouter-local-prod-messages/${release}-release.json`,
+    "utf8",
+  );
+  let json_data = JSON.parse(data);
+
+  // Add any message data with an id that does not already exist in data.json
+  for (let i = 0; i < json_data.length; i++) {
+    if (!json_result.find((x) => x.id === json_data[i].id)) {
+      json_result.push(json_data[i]);
```

Comment:
* the first version read always wins (which is fine if releases stay in decreasing order) - abort script if wrong order, comment near this code that it depends on this ordering
* the preview will always be of the latest released version, even though metrics will cover multiple (possibly all) versions (future epic)
* will need similar algorithm for merge looker data with this data and eventually remote settings data (can be part of spike PR - make note in spike PR)

Reply:
Instead of aborting script, I just ended up sorting the list in descending order

Reply:
Please add a comment that this will need to be checked once if/when we add dot-versions (129.0.1), since I'm not sure how sort's default collation order will handle that...

Reply:
I also had to update the script to start with a basically empty `data.json` (including the default about welcome message that we need to manually add) to account for when newer releases are being added so that they are prioritized first and not being overwritten.